accounts-db/src/accounts.rs

```diff
@@ -1319,8 +1328,104 @@ impl Accounts {
             durable_nonce,
             lamports_per_signature,
         );
+        let mut additional_lamports_result = None;
+        let create_dummy_accounts = true;
+        let mut pks = Vec::default();
+        let solana_vote_program: Pubkey = solana_vote_program::id();
+        if create_dummy_accounts {
+            let mut additional_lamports = 0;
+            let (_, us) = measure_us!({
+                for i in 0..accounts_to_store.len() {
+                    self.accounts_db.maybe_throttle_add();
+                    if accounts_to_store[i].1.owner() == &solana_vote_program {
+                        // vote programs will be stored on each slot and all dummys will all be duplicates
+                        continue;
+                    }
+                    let mut src_account = AccountSharedData::default();
+                    src_account.set_lamports(1_000_000_000);
+                    let mut pk = accounts_to_store[i].0.clone();
+                    let range = 4_000_000usize;
+                    let num_duplicates: usize = (900
```

Comment:
@HaoranYi set `num_duplicates` to 0 in theory to stop adding any dummy accounts. This lets us time max account growth rate under current normal conditions.